compose/ui/ui-graphics/src/skikoMain/kotlin/androidx/compose/ui/graphics/layer/SkiaGraphicsLayer.skiko.kt

```diff
@@ -265,6 +293,7 @@ actual class GraphicsLayer internal constructor() {
 
             if (clip || shadowElevation > 0f) {
```

Comment:
It wasn't in this PR, but why we check `shadowElevation > 0f`?

We don't check it in `RenderNodeLayer`

Reply:
Not sure here, it's part of initial Google's implementation

Reply:
https://youtrack.jetbrains.com/issue/CMP-6619/GraphicLayer.-Recheck-clipping-conditions